packages/eslint-config/.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
What are the arguments for disabling package-lock in `eslint-config`? We are enabling that feature everywhere else (or at least at the monorepo root level).

Reply:
It was copied before we turned on `package-lock.json`. Will remove it.

Reply:
See https://github.com/strongloop/loopback-next/pull/2579